docs/orders.rst

```diff
@@ -16,14 +17,31 @@ There are three possible delivery group statuses:
     The default status of newly created delivery groups.
 
 - ``SHIPPED``
-    The delivery group has been marked as shipped.
+    The delivery group has been marked as shipped by a shop operator. Only ``NEW`` groups can be marked as shipped.
 
 - ``CANCELLED``
-    The delivery group has been cancelled.
+    The delivery group has been cancelled by a shop operator. Both ``NEW`` and ``SHIPPED`` groups can be marked as cancelled.
+
+
+Delivery Group statuses flow
+----------------------------
+
+.. image:: img/delivery_group_statuses.png
+
+Regular flow of statuses is from ``NEW`` to ``SHIPPED``. For managing states saleor uses `django_fsm <https://github.com/kmmbvnr/django-fsm>`_ package. Actions performed within changing status are placed in Delivery Group model and are as follows:
```

Comment:
"Saleor" name should be capitalized here.

Reply:
When explicitely referring to models I would use their names from the codebase, so "DeliveryGroup" instead of "Delivery Group".

Reply:
Also "django-fsm" instead of "django fsm" seems more proper and is used in their documentation.